Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mono:latest
```

Comment:
@jaredbroad if I recall correctly we pinned to a certain mono version for performance reasons.